src/dotnet/commands/dotnet-build/BuildCommand.cs

```diff
@@ -29,6 +26,10 @@ public static BuildCommand FromArgs(string[] args, string msbuildPath = null)
         {
             var msbuildArgs = new List<string>();
 
+            if (args.Any(arg => arg.StartsWith("-verbosity:q") || arg.StartsWith("-verbosity:m"))) {
```

Comment:
This is likely using the wrong culture.

See: https://docs.microsoft.com/en-us/dotnet/api/system.string.startswith?view=netframework-4.7.2#System_String_StartsWith_System_String_

> This method performs a word (case-sensitive and culture-sensitive) comparison using the current culture.

(Unless there's something here that forces everything here to run in the invariant culture.)

Reply:
This should be ordinal. 

Also why startswith? We should probably handle the valid cases as exact match. Imagine if verbosity:maximal were added.

Reply:
@Eilon @nguerrera Updated.